eng/pipelines/common/templates/runtimes/send-to-helix-inner-step.yml

```diff
@@ -12,26 +12,26 @@ steps:
   # TODO: Remove and consolidate this when we move to arcade via init-tools.cmd.
```

Comment:
unrelated, what is this about?

Reply:
Not sure if this is the original intent of the comment but send-to-helix-step.yml exists in three versions: I created this particular file by extracting the Helix triggering logic from the CoreCLR version of the script in order to unify the Windows and Unix version; two other versions come from Arcade, they reside at:

https://github.com/dotnet/runtime/blob/master/eng/common/templates/steps/perf-send-to-helix.yml
https://github.com/dotnet/runtime/blob/master/eng/common/templates/steps/send-to-helix.yml

There's not much difference among them and they could be easily unified but I believe that avoiding the code duplication between Windows and Unix requires using a subordinate script (like I did in CoreCLR) and my proposal to take the change to Arcade was not met with too much enthusiasm, see here:

https://github.com/dotnet/arcade/pull/6227

Reply:
To me it seems like we can unify them already? We could use the script extension that we set in `xplat-setup` and just use single hyphen arguments which are supported, right?

Reply:
I think you're right that we should be able to unify the Windows and Unix variants in <code>send-to-helix-inner-step.yml</code>. Sadly we cannot use the existing extension switch in xplat-setup though as the Windows variant uses a PowerShell, not a batch script (ps1 instead of cmd) but that's not rocket science to add to xplat-setup as another variant extension. There are also a few additional subtle differences (-ci vs. --ci) hitting the traditional annoyance that there's no easy way to write a conditional expression in YAML scripts and the "step" template doesn't support variable definitions.

Reply:
Maybe we can just add support for single hyphen arguments in the `msbuild.sh` script as @ViktorHofer did a while ago for the `build.sh` script in arcade. This was the exact reason why we added that support, to be able to have commands that work on both Unix and Windows scripts:

https://github.com/dotnet/runtime/blob/8bc983405f3fb900da8ba77eb1d02e7ceb9a9a4e/eng/common/build.sh#L84